static/assets/js/entity-reference/app.js

```diff
@@ -0,0 +1,59 @@
+(function(){
+    "use strict";
+
+    var app = angular.module('entityReferenceApp', []);
+
+    app.controller('EntityReferenceCtrl', ['$http', EntityReferenceCtrl]);
+
+    app.component('entityReference', {
+        templateUrl: '/assets/views/entities.html',
+        controller: 'EntityReferenceCtrl as ctrl',
+        bindings: {
+            title: '@',
```

Comment:
not sure where my other comment went, but what are your thoughts on prefixing the attributes with "bb-" or something so they don't conflict w/ html attributes (i.e. "title" might conflict).